src/gsheets/sync_gsheets_active_proposal_votes_snapshot.js

```diff
@@ -439,7 +389,7 @@ const syncGSheetsActiveProposalVotes = async (
 
   // Output the round summary
   proposalSummary = await calculateProposalSummary(
-    activeProposals,
+    activeProposals, // ACTIVE PROPOSALS IS NOT DEFINED??
```

Comment:
Is this comment valid? Is this a TODO?

Reply:
Calling that function [here](https://github.com/oceanprotocol/DAOBot/blob/f284abf8d1c6a122e1238422225546de35d7421e/src/scripts/update_funding_round.js#L300) returns an error, I added it as a note.

Reply:
Thank you!